spec/index.html

```diff
@@ -1498,6 +1498,60 @@ <h2>RDF Datasets</h2>
     between datasets would require RDF graph entailments between the graphs with the same name
     (adding in empty graphs as required).</p>
 
+  <table>
+    <caption>Definition of an RDF dataset</caption>
+    <tr>
+       <td class="semantictable">
+       <p>An <dfn data-lt="RDF dataset">RDF dataset</dfn> I is a set:</p>
+              { G, (&lt;u<sub>1</sub>&gt;, G<sub>1</sub>), (&lt;u<sub>2</sub>&gt;, G<sub>2</sub>), . .
+              . (&lt;u<sub>n</sub>&gt;, G<sub>n</sub>) }<br>
+              where n&ge;0 and G and each G<sub>i</sub> are graphs, and each &lt;u<sub>i</sub>&gt; is an IRI. Each
+              &lt;u<sub>i</sub>&gt; is distinct.</p>
+            <p>G is called the default graph. (&lt;u<sub>i</sub>&gt;, G<sub>i</sub>) are called named
+              graphs.</p>
+        </td>
+    </tr>
+  </table>
+  <table>
+    <caption>Definition of active graph in an RDF dataset</caption>
+    <tr>
+       <td class="semantictable">
+            <p>The <dfn data-lt="active graph">active graph</dfn> in an RDF dataset is the graph from the dataset used for basic graph pattern
+              matching.</p>
+       </td>
+    </tr>
+  </table>
```

Comment:
This is SPARQL specific and can be left in SPARQL Query. It changes during query execution (GRAPH).

Reply:
Done